README.md

```diff
@@ -11,6 +11,13 @@ A user with sudo privilidges. `git` needs to be installed to clone the repo. Mos
 
 The `install-gns3.sh` script will automate most of the installation process.
 
+My setup, for which the script works smoothly:
+- 5.5.16-1-MANJARO
+- python 3.8
+- pip 20.0.2
+- aur/python-pypi2pkgbuild-git 0.1.r95.g8a8e1f0-1
+- gns3 v2.2.7
+
```

Comment:
Specifying a kernel version + package version on a rolling release distro does not seem to be necessary as those would change quite often. I test the script on vanilla Arch as well. 

A good compromise could be something like:

### Examples of working setups

Confirmed by @LaumiH to work with the following setup:

> My setup, for which the script works smoothly:
> - 5.5.16-1-MANJARO
> - python 3.8
> - pip 20.0.2
> - aur/python-pypi2pkgbuild-git 0.1.r95.g8a8e1f0-1
> - gns3 v2.2.7